Helper/Order.php

```diff
@@ -1170,44 +1180,7 @@ protected function validateCaptureAmount(OrderInterface $order, $captureAmount)
         $isInvalidAmountRange = $order->getTotalInvoiced() + $captureAmount > $order->getGrandTotal();
 
         if($isInvalidAmount || $isInvalidAmountRange) {
-            throw new Exception( __('Capture amount is invalid'));
+            throw new \Exception( __('Capture amount is invalid'));
```

Comment:
is slash needed?

Reply:
In this name space `namespace Bolt\Boltpay\Helper;` and the removal of the previous `use` on line 46, the slash is needed to denote top level namespace.  This style is the accepted norm in OO PHP.